[FDP-623] Geen dimValue teruggeven in GetStatusResponse als relais uit staat (voor zowel SSLD als LMD)

jasperkamerling opened the pull request.

kevinsmeets committed:
#242 ~ Don't map the dim value if relay state is off.

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

kevinsmeets committed:
Refactors code to fix failing test case.

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

jasperkamerling committed:
FDP-623: Use null dim value when light is off

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

jasperkamerling committed:
FDP-623: Use null dim value when light is off

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

jasperkamerling committed:
FDP-623: Update cucumber test to match dim functionality

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

JelleHoffman commented on integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature:
```diff
@@ -1,4 +1,4 @@
-@PublicLighting @Platform @PublicLightingSetLightSchedule
+@PublicLighting @Platform @PublicLightingSetLightSchedule @Jasper
```
Development code?

jasperkamerling replied:
oops, is nu weer weg

jasperkamerling committed:
FDP-623 Remove development test tag

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

smvdheijden commented on osgp/platform/osgp-adapter-ws-publiclighting/src/main/java/org/opensmartgridplatform/adapter/ws/publiclighting/application/mapping/converters/LightValueConverter.java:
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Alliander N.V.
```
```suggestion
 * Copyright 2022 Alliander N.V.
```

smvdheijden commented on osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/application/mapping/LightValueToOslpLightValueMapper.java:
```diff
@@ -28,9 +28,9 @@ public LightValueDto convertFrom(
 
-    int dimValue = 0;
+    Integer dimValue = null;
     if (source.hasDimValue()) {
```
```suggestion
    if (source.hasDimValue() && source.hasOn()) {
```

smvdheijden reviewed (changes requested):
- Looks like this pull request contains several formatting changes in the feature files. Check if you could keep numbers right-aligned. In Eclipse this could be set in the preferences, not sure about IntelliJ.
- You've added some additional examples to the SetLightSchedule feature, but shouldn't there be any additional examples in the GetLightStatus feature as well?

jasperkamerling commented:
Intellij doesnt have a different alignment for numbers. But the eclipse allignement also doesn't really work because most of the numbers here are actually lists of numbers so those would need to go on the left.
I'll change them back for now but maybe we can look into it later and fix this in Eclipse.

jasperkamerling committed:
FDP-623: Process review remarks

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

jasperkamerling committed:
FDP-623: Add ws-secret-mangement to da profile

The osgp-secret-management profile requires the osg-ws-secret-mangement module.
The osm module is added with the da profile but the owsm module isn't.
To fix this we need to import the owsm module with the da profile.

Signed-off-by: Jasper Kamerling <jasper.kamerling@alliander.com>

smvdheijden commented on osgp/shared/pom.xml:
```diff
@@ -134,6 +134,7 @@
       <modules>
         <module>osgp-kafka-logger</module>
         <module>osgp-ws-distributionautomation</module>
+        <module>osgp-ws-secret-management</module>
```
osgp-ws-secret-management should not be needed here.

smvdheijden committed:
FDP-623 ~ Removes secret management from da profile

Signed-off-by: Sander van der Heijden <sander.van.der.heijden@alliander.com>
